client/package.json

```diff
@@ -19,16 +24,16 @@
     "react-scripts": "^3.4.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "dotenv -e .env -- cross-var cross-env PORT=%CLIENT_PORT% react-scripts start",
```

Comment:
We can use our npm scripts to have a universal environment setup and teardown. Using these libraries (dotenv-cli, cross-var, and cross-env) allows us to set the environment variables in a single source (.env files) and then run the environment setup files with a single command. The syntax used from cross-var hopefully also makes this work for Windows environments as well as Mac. 

Useful Links:
https://medium.com/@arrayknight/how-to-use-env-variables-in-package-json-509b9b663867

https://laravel-news.com/using-variables-in-your-env-file

https://stackoverflow.com/questions/40714583/how-to-specify-a-port-to-run-a-create-react-app-based-project

https://create-react-app.dev/docs/adding-custom-environment-variables/